test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -110,11 +110,14 @@ abstract class LorisIntegrationTest extends TestCase
         $user->updatePassword($password);
 
         $dev          = $this->config->getSettingFromXML('dev');
-        $selenium_url = isset($dev['selenium_url']) ? $dev['selenium_url'] : 'http://localhost:4444/wd/hub';
+        $selenium_url = $dev['selenium_url']) 'http://localhost:4444/wd/hub';
```

Comment:
line 113 looks to be missing ??

Reply:
Thanks, weird that that didn't get caught by `make checkstatic`.